### ISSUE TITLE
database is self destruction ?

### ISSUE THREAD

**Issue by leaderdog** [jellyfin/jellyfin#4438]
I decided to try native mode, add network location to the current library (optional location).  It wouldn't save it.
So I created a new folder on a separate drive called it movies2 and put a single movie in there.  It still didn't accept the network location in the option location.
Out of frustration I hit save a bunch of times and it apparently ended up making movies21 movies211 movies 2111 etc to movies 211111.
I noticed this when I was trying to get mkv shim to work and wasn't sure why the heck there was so many movie folders.
so I deleted them from the server as well as two existing folders I don't use, comics and books (they didn't work how I wanted).
Watched a show that night, all was good, via kodi on my htpc.
Next morning go to do exercises and Kodi says processing 56000 updates.
I didn't know why.  That didn't make sense.  checked a show and nothing worked.
I shut Kodi off.
came to my pc and checked jellyfin through the web access and there are 0 movies and 0 tv shows in there respective folders.
Fortunately, I have a backup of my jellyfin files so I copied them back over to where they belong and everything is back including the movies211 etc.
all my shows and movies are listed just fine in jellyfin and kodi on the other computer is happy as well.
Later I deleted the folders again and the exact same thing happened, all the movies and shows missing from jellyfin.
copied them back from my backups, all good again.
deleted each unwanted folder one at a time and restarted jellyfin in between each and it seemed to work.  All was good.
I just finished watching a movie on kodi and when it went back to the main screen it showed it was doing updates to the tune of a ridiculous amount.
Shut off Kodi.
Went to my pc to check Jellyfin, sure enough nothing is in the movies or tv folders again.
copied the backups over again but this time I had copied the files that had the removed folders.  Which I now think will not hold up.

Why is Jellyfin deleting my movies and tv shows?
Why can't I delete folders that aren't wanted without Jellyfin self destruction the database?
What should I be doing to stop this. I really don't want to have to start over and retag all the watched shows and movies.  I went this root because Kodi by itself would constantly screw up and lose my database.  I was hoping I was done with that.  This is very peculiar.

Please any assistance would be wonderful.

All Machines are windows 10.
latest kodi 18.6 (?)
Jellyfin 10.0.6 (latest version)

As I was typing this Jellyfin was in the web browser and all movies/tv shows from watched list disappeared.  I was left with just the headers for Movies and Tv Shows as well as a few movies I started but didn't finish.

Here is a link to my log.  I'm going to put my old backups on with the multiple movie2 folders on since it was at least stable with that.
https://www.dropbox.com/s/5ts4gytvz7rarlr/log_20201107.log?dl=0

**Comment by crobibero** [comment]
There's only one path Jellyfin has to actually delete media from disk, and I don't see that function being logged in your logfile, so I think you might have a non-jellyfin issue.

**Comment by leaderdog** [comment]
I'm not deleting media, I'm removing folders in Jellyfin.

All I'm doing is using firefox to log into my jellyfin admin account and removing the folders via the options Jellyfin supplies.  Not sure how anything should go wrong from there.?  It certainly shouldn't cause Jellyfin to remove the database?  or remove my media from Jellyfin.  Like I said I can access the files if they're in the continue watching section, but everything else disappears.

Is this a case where I have to start all over and remark all the series and movies?  So far today there hasn't been any issues with Jellyfin with the old backup on with the multiple movies21 etc folders.  I deleted the movies2 folder a while back.  So it's literally directed to nothing.
I just looked out of curiousity and each of the movies21 211 2111 etc folders all point to the same place L:\Movies2 .  Which as I said, doesn't exist on the drive anymore

**Comment by crobibero** [comment]
What do you mean by removing the folders?
Are you removing folders from the library?

**Comment by leaderdog** [comment]
correct.  Just removing the extra folders that aren't used in Jellyfin.
I have Movies and TV Shows.  I deleted everything else comics, books, and Movies2 (which was just for a test) and it's multiple duplicates.  Then Jellyfin later the same day just removed all the files in both Movies and TV Shows. It worked all day until the evening and then it emptied the database...

**Comment by leaderdog** [comment]
I did some googling and found some info that didn't work for me, but when I was looking through the log it kept pointing to:
`C:\ProgramData\Jellyfin\Server\root`
which I didn't have.  So I just added it and copied the folder/files from:
`C:\Users\User\AppData\Local\Jellyfin\root`
So far the database hasn't self destructed.  

Kodi is constantly spinning "Jellyfin for Kodi 0%  Removing library"

I did a repair library under the addon and all my movies and shows are there, but the "removing Library" remains.

I then exited Kodi, and restarted it to see it was all good, switched to the other profile to update it as well, it was all good.  Went back to mine and again the Removing library 0% spinning in the corner.

Not sure how to resolve that, but at least it didn't kill my database this time around.

So progress with issues :)
